etna/analysis/eda_utils.py

```diff
@@ -19,7 +18,7 @@
 plot_pacf = sm.graphics.tsa.plot_pacf
 
 
-def cross_corr_plot(ts: "TSDataset", n_segments: int = 10, maxlags: int = 21, segments: Optional[Sequence] = None):
+def cross_corr_plot(ts: "TSDataset", n_segments: int = 10, maxlags: int = 21, segments: Sequence = None):
```

Comment:
Is this normal? I thin we have Optional in such case in a lot of places.

Reply:
Fixed